### PR TITLE
[KEYCLOAK-11193] Change 'disableLogging' to 'enableLogging' and default to false

### PR DESCRIPTION
**Pull request by jonkoops**
See: https://github.com/keycloak/keycloak/pull/6270

### CODE REVIEW

**Comment by stianst** [comment]
https://github.com/keycloak/keycloak-documentation/pull/723

**Review by mhajas — DISMISSED** [review]

**Review by mhajas — DISMISSED** [review]

**Comment by abstractj** [comment]
@jonkoops thank you, changes are looking good. I'm trust triggering Travis again to see how it goes. The failures are completely unrelated with your change.

File: adapters/oidc/js/src/main/resources/keycloak.js [hunk 1]
```diff
@@ -163,8 +163,10 @@
                     kc.pkceMethod = initOptions.pkceMethod;
                 }
 
-                if (typeof initOptions.disableLogging === 'boolean') {
-                    kc.disableLogging = initOptions.disableLogging;
+                if (typeof initOptions.enableLogging === 'boolean') {
+                    kc.enableLogging = initOptions.enableLogging;
+                } else {
+                    ks.enableLogging = false;
```
Comment (abstractj) [inline adapters/oidc/js/src/main/resources/keycloak.js]: @jonkoops @mhajas I believe this is the reason why tests are failing on Travis. We need to replace `ks` by `kc`.

**Review by abstractj — CHANGES_REQUESTED** [review]
Please, see my comments above.

File: adapters/oidc/js/src/main/resources/keycloak.js (hunk 1 above)
Comment (jonkoops) [reply adapters/oidc/js/src/main/resources/keycloak.js]: Ah, nice. I was already going through the logs trying to find the problem. Should be fixed now.

**Review by mhajas — APPROVED** [review]

**Review by abstractj — APPROVED** [review]

**Pull request merged**